bindings/core/tests/secrets_manager.rs

```diff
@@ -25,3 +25,24 @@ async fn generate_addresses() -> Result<()> {
 
     Ok(())
 }
+
+#[tokio::test]
+async fn generate_evm_addresses() -> Result<()> {
+    let secret_manager = SecretManager::try_from_mnemonic(
+        "endorse answer radar about source reunion marriage tag sausage weekend frost daring base attack because joke dream slender leisure group reason prepare broken river",
+    )?;
+
+    let method = SecretManagerMethod::GenerateEvmAddresses {
+        options: GetAddressesOptions::default(),
+    };
+
+    let response = secret_manager.call_method(method).await;
+    match response {
+        Response::GeneratedEvmAddresses(addresses) => {
+            assert_eq!(addresses[0], "0xbfc69bd510a9a93449d31b99d23dbc55ecf1f499")
```

Comment:
Can you make it generate `0xcaEfDe2b487DED55688765964320Ff390cd87828` with coin type 60? Because that's what metamask does and what most people also want to get
Also we need a way to specify the hardened/non hardened part, for Ed25519 we just always make all values hardened so there it wasn't needed, but for Ethereum it's mixed by default like this: `m/44'/60'/0'/0/0` the internal and address index values aren't hardened, this would then be `0xcaEfDe2b487DED55688765964320Ff390cd87828`
Right now it does `m/44'/4219'/0'/0'/0'`